### PR TITLE
update ndc-postgres to v0.6.0

### PR DESCRIPTION
**Pull request by soupi**
https://github.com/hasura/ndc-postgres/releases/tag/v0.6.0

### CODE REVIEW

File: scripts/update-ndc-postgres.sh
Comment (soupi) [inline scripts/update-ndc-postgres.sh:1]: Added a script to make this update easier in the future.

File: scripts/update-ndc-postgres.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
```
Comment (SamirTalwar) [inline scripts/update-ndc-postgres.sh:2]: I suggest failing fast.

```suggestion

set -e
set -u
set -o pipefail

```

File: scripts/update-ndc-postgres.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    'neon' \
+    'postgres' \
+    'postgres-alloydb' \
+    'postgres-azure' \
+    'postgres-cosmos' \
+    'postgres-gcp' \
+    'postgres-timescaledb' ; do
+
+# add new version
+cat registry/"${variant}"/metadata.json | jq \
```
Comment (SamirTalwar) [inline scripts/update-ndc-postgres.sh:28]: `cat $FILE | jq $QUERY` can be simplified to `jq $QUERY $FILE`.

File: scripts/update-ndc-postgres.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```
Comment (SamirTalwar) [inline scripts/update-ndc-postgres.sh:1]: Bash doesn't always live in `/bin`.

```suggestion
#!/usr/bin/env bash
```

**Review by SamirTalwar — APPROVED** [review]

**Comment by soupi** [comment]
Thanks for the review @SamirTalwar !

**Pull request merged**